app/models/topic_embed.rb

```diff
@@ -27,7 +27,7 @@ def self.imported_from_html(url)
   end
 
   # Import an article from a source (RSS/Atom/Other)
-  def self.import(user, url, title, contents)
+  def self.import(user, url, title, contents, category_id = nil)
```

Comment:
This method has quite a few parameters already. Could I ask that we use a keyword argument for it instead? It really helps when examining the call signature.